Tools/LambdaTestTool-v2/README.md

```diff
@@ -0,0 +1,288 @@
+    - [New Features](#new-features)
+- [Getting help](#getting-help)
+- [Installing](#installing)
+- [Running the Test Tool](#running-the-test-tool)
+    - [Lambda Emulator Mode](#lambda-emulator-mode)
+    - [API Gateway Emulator Mode](#api-gateway-emulator-mode)
+        - [Required Configuration](#required-configuration)
+    - [Combined Mode](#combined-mode)
+- [Command Line Options](#command-line-options)
+- [API Gateway Configuration](#api-gateway-configuration)
+    - [Single Route Configuration](#single-route-configuration)
+    - [Multiple Routes Configuration](#multiple-routes-configuration)
+- [Example Lambda Function Setup](#example-lambda-function-setup)
+    - [1. Lambda Function Code](#1-lambda-function-code)
+    - [2. Configuration Files](#2-configuration-files)
+    - [3. AWS_LAMBDA_RUNTIME_API](#3-aws_lambda_runtime_api)
+    - [4. API Gateway Configuration](#4-api-gateway-configuration)
+    - [5. Testing the Function](#5-testing-the-function)
+
+
+## Getting help
+
+This tool is currently in preview and there are some known limitations. For questions and problems please open a GitHub issue in this repository.
+
+## .NET Aspire integration
+The easiest way to get started using the features of the new test tool is with .NET Aspire. The integration takes care of installing the tool and provides .NET Aspire extension methods for configuring your Lambda functions and API Gateway emulator in the .NET Aspire AppHost. It avoids all steps list below for installing the tooling and setting up environment variables.
+
+Check out the following tracker issue for information on the .NET Aspire integration and steps for getting started. https://github.com/aws/integrations-on-dotnet-aspire-for-aws/issues/17
+
+## Installing
+
+The tool is distributed as .NET Global Tools via the NuGet packages. To install the tool execute the following command:
+
+```
+dotnet tool install -g amazon.lambda.testtool --prerelease
+```
+
+To update the tool run the following command:
+
+```
+dotnet tool update -g amazon.lambda.testtool --prerelease
+```
+
+## Running the Test Tool
+
+### Lambda Emulator Mode
+Use this mode when you want to test Lambda functions directly without API Gateway integration.
+
+```
+# Start Lambda emulator on port 5050
+dotnet lambda-test-tool start --lambda-emulator-port 5050
+```
+
+### API Gateway Emulator Mode
+Use this mode when you want to test Lambda functions through API Gateway endpoints. This mode requires additional configuration through environment variables.
+
+```
+# Start API Gateway emulator on port 5051 in REST mode
+dotnet lambda-test-tool start \
+    --api-gateway-emulator-port 5051 \
+    --api-gateway-emulator-mode Rest
+
+```
+#### Required Configuration
+When running via command line, you must set the environment variable for API Gateway route configuration:
+
+Linux/macOS:
+```bash
+export APIGATEWAY_EMULATOR_ROUTE_CONFIG='{"LambdaResourceName":"AddLambdaFunction","HttpMethod":"Get","Path":"/add/{x}/{y}"}'
```

Comment:
The command you gave above does not run the Lambda client so you need to add the endpoint to the env vars where the client is running.